packages/core/lib/configAdapter.js

```diff
@@ -0,0 +1,103 @@
+function sanitizeGanacheOptions(ganacheOptions) {
```

Comment:
This function name doesn't convey what the function does, which is operating on `network_id`. It made sense in previous revisions, and now that you're settling on a new approach, this could be refactored. Instead of:

```javascript
const sanitizedG = sanitizeGanacheOptions(config);
```
consider this:

```javascript
config.network_id = resolveNetworkId(config.network_id);
```

Reply:
Yes, this makes sense. I will refactor this. Thanks @cds-amal .